src/test/java/org/jabref/model/entry/EntryLinkListTest.java

```diff
@@ -0,0 +1,38 @@
+package org.jabref.model.entry;
+
+import org.jabref.model.database.BibDatabase;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+
+public class EntryLinkListTest {
+
+    String fieldValue = "test";
+    BibDatabase bibDatabase = new BibDatabase();
+
+    @Test
+    public void givenFieldValueAndDatabaseWhenParsingThenExpectLink() {
+        List<ParsedEntryLink> links = EntryLinkList.parse(fieldValue, bibDatabase);
+        ParsedEntryLink link = links.get(0);
+
+        assertEquals(fieldValue, link.getKey());
```

Comment:
Please use only one assert per test, make it easier to spot erros on failing tests and I would recommend directly comparing the ParsedEntryLink: Just create a new ParsedEntrylink with the expected values and compare it. 
You may need to implement/override (or let it generate from your IDE) a toString() method, otherwise your junit test will probably fail (Encountered that recently in another project)